api/v2/views/provider_type.py

```diff
@@ -2,6 +2,7 @@
 
 from api.v2.serializers.details import ProviderTypeSerializer
 from api.v2.views.base import AuthViewSet
+from api.v2.views.mixins import MultipleFieldLookup
```

Comment:
It doesn't look like `MultipleFieldLookup` was used as a mixin for `PlatformTypeViewSet`. 
Should we pull out the import then?